### ISSUE TITLE
The documentation has broken links from external sites

### ISSUE THREAD

**Issue by gaaclarke** [flutter/website#7754]

**Comment by Hixie** [comment]
@gaaclarke We want to avoid having more than one (well, two, if you count the master branch API docs) version of the API docs because we noticed a huge antipattern in lots of other SDKs wherein they have lots and lots of versions of the API docs and people often end up looking at old versions (e.g. due to following links or search engines not noticing they're old or whatever).

@itsjustkevin I'm not sure this belongs in flutter/website, it's about the API docs not the docs.flutter.dev site.

In the absence of a better solution, can we just get the 404 logs and regularly set up redirects?

**Comment by itsjustkevin** [comment]
@Hixie thanks for pointing this out.  Do we want to move it back, and do we have someone who wants to take a look at this issue?

cc: @atsansone

**Comment by sfshaza2** [comment]
No, sorry, this absolutely belongs in the flutter/flutter repo.  :)
